manifests/init.pp

```diff
@@ -6,9 +6,11 @@
 
   # JVM Settings
   $javahome                                                      = undef,
+  Jira::Jvm_types $jvm_type                                      = 'openjdk-11',
```

Comment:
Please do not use a custom type here as it is only used in one class and is a very simple list.

Reply:
Did so, because same thing was done in Jira module, but now i reverted it.